base/inference.jl

```diff
@@ -2965,7 +2965,8 @@ function inlining_pass(e::Expr, sv, linfo)
     # by the interpreter and inlining might put in something it can't handle,
     # like another ccall (or try to move the variables out into the function)
     if is_known_call(e, Core.Intrinsics.ccall, sv)
-        i0 = 5
+        # 4 is rewrite to 2 below to handle the callee.
```

Comment:
do you mean rewritten? ([ci skip] strictly typo-fix commits if you've verified it passes `make check-whitespace` locally)